code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm

```diff
@@ -61,16 +61,12 @@
 	var/obj/item/weapon/tome/A = null
 	A = user.get_active_hand()
```

Comment:
Remove this, the tome = istype(A), the if(!tome), everything held within that if(!tome), and replace it with var/obj/item/weapon/tome/A = locate() in user.held_items